src/css/graph.css

```diff
@@ -143,7 +143,7 @@
 }
 .rickshaw_graph .y_ticks text,
 .rickshaw_graph .x_ticks_d3 text {
-	opacity: 0.5;
+	fill: rgba(0, 0, 0, 0.5);
```

Comment:
Text render slightly brighter than solid fills due to it's thin lines and curve smoothing. I'd suggest a slightly darker value of `rgba(0, 0, 0, .65)`.